config/default.json

```diff
@@ -36,5 +36,6 @@
 		"HOST_ONLY": true,
 		"SECURE": true,
 		"EXPIRES_SECONDS": 2592000000
-	}
+	},
+	"DISPLAY_INCIDENTS_IN_MIN": 2880
```

Comment:
not sure is needed, unleless there is a good reason to have a diferent value compared to default.schema